### PR TITLE
fix(kicon): standardize kicon svgs

### PR DESCRIPTION
**Pull request by adamdehaven**
### Summary

Update all `KIcon` SVGs to have the same default color.

#### Changes made:
* All icons now have a default fill of `#A3BBCC`

### PR Checklist
- [x] Does not introduce dependencies
- [x] **Functional:** all changes do not break existing APIs and if so, bump major version.
- [x] **Tests pass:** check the output of yarn test packages/<Kongponent>
- [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
- [x] **Framework style:** abides by the essential rules in Vue's style guide
- [x] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs), or leftover comments
- [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: packages/KIcon/icons/icn-brain.svg [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <svg width="32" height="32" viewBox="-2 -2 28 28" xmlns="http://www.w3.org/2000/svg">
-  <path fill="#A4BBCB" d="M11 4h2v16h-2z"/>
-  <path fill="#A4BBCB" d="M6 8h12v2H6zM2 14h20v2H2zM6 20h12v2H6z"/>
-  <path d="M12 4a2 2 0 100-4 2 2 0 000 4z" stroke="#A4BBCB" stroke-width="2" fill="#FFF" />
-  <path d="M0,14.75a1.5,1.5 0 1,0 3,0a1.5,1.5 0 1,0 -3,0" stroke="#A4BBCB" stroke-width="2" fill="#FFF" />
-  <path d="M21,14.75a1.5,1.5 0 1,0 3,0a1.5,1.5 0 1,0 -3,0" stroke="#A4BBCB" stroke-width="2" fill="#FFF" />
-  <path d="M5.25 10.25a1.5 1.5 0 100-3 1.5 1.5 0 000 3zM18.75 10.25a1.5 1.5 0 100-3 1.5 1.5 0 000 3zM5.25 22.25a1.5 1.5 0 100-3 1.5 1.5 0 000 3zM18.75 22.25a1.5 1.5 0 100-3 1.5 1.5 0 000 3z" stroke="#A4BBCB" stroke-width="2" fill="#FFF" />
+  <path fill="#A3BBCC" d="M11 4h2v16h-2z"/>
```
Comment (chrisvdp) [inline packages/KIcon/icons/icn-brain.svg:2]: Can you add a title tag while we're here?

**Review by chrisvdp — CHANGES_REQUESTED** [review]

File: packages/KIcon/icons/icn-brain.svg (hunk 1 above)
Comment (adamdehaven) [reply packages/KIcon/icons/icn-brain.svg:2]: @chrisvdp so I'm going through all the SVGs -- should all have a title? I know a11y the answer is yes, but, for something like `icon-check` would the title just be "Checkmark" or something better?

Comment (chrisvdp) [reply packages/KIcon/icons/icn-brain.svg:2]: I think at least a basic title is a good backstop for when consumers don't add their own.

Comment (adamdehaven) [reply packages/KIcon/icons/icn-brain.svg:2]: Comment for transparency: titles are auto-generated within `KIcon` if they do not exist.

**Review by kaiarrowood — APPROVED** [review]
This looks good to me!

**Pull request merged**